indexer-app/src/cmr/indexer/data/concepts/collection/keyword.clj

```diff
@@ -89,6 +89,10 @@
         related-url-urls (map :URL related-urls)
         related-url-titles (map :Title related-urls)
         related-url-descriptions (map :Description related-urls)
+        ;; Pull author info from both creator and other citation details
+        authors (concat
```

Comment:
Did you want to also thread this to `(map str/trim)`? If so, it's probably worth breaking that into its own function somewhere